developer-model-service/catalog-info.yaml

```diff
@@ -0,0 +1,2036 @@
+# Adapted from: https://raw.githubusercontent.com/benwilcock/backstage-catalogs/main/ai-asset-catalog.yml
+####################################
+# COMPONENTS
+# A Component represents a model server that's deployed, containing metadata about it
+####################################
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: developer-model-service
+  annotations:
+    backstage.io/techdocs-ref: dir:./
+  description: 'vLLM and 3scale-based model service providing models for developer tools.'
+  links:
+    - url: https://model-service.apps.domain.com
+      title: Access
+      type: website
+      icon: WebAssett
+    - url: https://ibm-granite-8b-code-instruct-vllm.apps.domain.com
+      title: API URL
+      type: website
+      icon: WebAsset
+  tags:
+    - genai
+    - ibm-granite
+    - vllm
+    - llm
+    - developer-model-service
+    - authenticated
+    - gateway
+spec:
+  type: model-server
```

Comment:
just for my own sanity @johnmcollier this is the `type` (vs. `kind`), correct?

Reply:
Yup, this is the custom `type` we'll be using in lieu of a custom kind, (and same applies for the `Resource` and `API` objects).